loafwallet/AppDelegate.swift

```diff
@@ -46,47 +46,69 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         return true
     }
-
+    
     func applicationDidBecomeActive(_ application: UIApplication) {
         UIApplication.shared.applicationIconBadgeNumber = 0
     }
-
+    
     func applicationWillEnterForeground(_ application: UIApplication) {
         applicationController.willEnterForeground()
     }
-
+    
     func applicationDidEnterBackground(_ application: UIApplication) {
         applicationController.didEnterBackground()
     }
-
+    
     func applicationWillResignActive(_ application: UIApplication) {
         applicationController.willResignActive()
     }
-
+    
     func application(_ application: UIApplication, performFetchWithCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
         applicationController.performFetch(completionHandler)
     }
-
+    
     func application(_ application: UIApplication, shouldAllowExtensionPointIdentifier extensionPointIdentifier: UIApplicationExtensionPointIdentifier) -> Bool {
         return false // disable extensions such as custom keyboards for security purposes
     }
-
+    
     func application(_ app: UIApplication, open url: URL, options: [UIApplicationOpenURLOptionsKey : Any] = [:]) -> Bool {
         return applicationController.open(url: url)
     }
-
+    
     func application(_ application: UIApplication, shouldSaveApplicationState coder: NSCoder) -> Bool {
-       return true
+        return true
     }
     
     func application(_ application: UIApplication, shouldRestoreApplicationState coder: NSCoder) -> Bool {
         return true
     }
     
+    
+    /// Sets the correct plist file
     private func setFirebaseConfiguration() {
-        //Conditional to pass the CircleCI checks
-        if Bundle.main.path(forResource: "GoogleService-Info", ofType: "plist") != nil {
-            FirebaseApp.configure()
+        
+        var filePath: String = ""
+        
+        #if Release
+            guard let releasePath = Bundle.main.path(forResource: "file-resource/release/GoogleService-Info", ofType: "plist") else { return
```

Comment:
@kcw-grunt the resource file is the same for both `release` and `debug`.  is this what you really want?

Also are you sure about the path? It seems like we should have `file-resources` instead of `file-resource`

Reply:
let me check.

Reply:
Wow....full marks.  This was wrong. Thanks. 👀🎉

Reply:
Haha 
Np

Reply:
I reconfigured @mosadialiou

Reply:
yes @mosadialiou